tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "compilerOptions": {
+        "outDir": "./dist",
+        "target": "ES2020",
+        "moduleResolution": "node",
+        "esModuleInterop": true
+    },
+    "include": ["./src/*", "src/declarations/interfaces.d.ts", "src/declarations/types.d.ts"]
```

Comment:
why not change include and exclude to this:
`"include": ["src/**/*"],
"exclude": ["node_modules", "**/*.spec.ts"]`
which is the TypeScript recommended include exclude settings